starknet_py/net/account/account_client.py

```diff
@@ -629,7 +629,7 @@ async def _verify_message_hash(self, msg_hash: int, signature: List[int]) -> boo
             nonce=None,
         )
         try:
-            await self.call_contract(invoke_tx=invoke_tx)
+            await self.call_contract(invoke_tx=invoke_tx, block_hash="latest")
```

Comment:
Why was this added?

Reply:
Without it, full node client had a problem in rpc_call as devnet does not support other blocks than latest.